dist/bundle.js

```diff
@@ -362,6 +369,14 @@ return /******/ (function(modules) { // webpackBootstrap
 	    }, settings);
 	  }
 
+	  function _authWithOAuthToken(provider, token, fn, settings) {
```

Comment:
The 'token' and 'fn' arguments should switch places:
function _authWithOAuthToken(provider, fn, token, settings) {...}